### PR TITLE
feat: add account form component

### LINKED ISSUE #22: Create AccountForm component
## Figma Reference
https://www.figma.com/file/uEckQY4EaQ5rXwyISg6d1g/Fuel-Wallet?node-id=290%3A16150

## Props
- `account?: Account`

## Form fields
- `name: String`

### PR DESCRIPTION
**Pull request by matt-user**
closes #22

### CODE REVIEW

File: packages/app/src/systems/Account/components/AccountForm/AccountForm.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          <HelperIcon message="The name for your new account">
+            Account Name
+          </HelperIcon>
+        }
+        render={({ field }) => (
+          <Input>
+            <Input.Field
+              {...field}
+              aria-label="Account Name"
+              placeholder="Type your name"
```
Comment (LuizAsFight) [inline packages/app/src/systems/Account/components/AccountForm/AccountForm.tsx:31]:
```suggestion
              placeholder="Type account name"
```

**Review by LuizAsFight — DISMISSED** [review]
can we create a really simple test for this new component?

**Comment by LuizAsFight** [comment]
@matt-user this PR has broken build which makes harder to quick testing
we normally review only really ready for review PRs, by that means `all green`


if build is broken in CI , normally you can reproduce locally with `pnpm build:all`

**Review by luizstacio — APPROVED** [review]
LGTM

**Pull request merged**